app/mailers/notifier.rb

```diff
@@ -0,0 +1,10 @@
+class Notifier < ActionMailer::Base
+  def statistic statistic
+    @daily_statistic = statistic
+
+    mail subject: t('digest.email_header.subject', date: @daily_statistic.for_today),
+         to:      User.all.map {|user| user.email},
+         bcc:     ENV['BCC'],
```

Comment:
@vlasiak, Can you please introduce settings so we could talk to `Settings` object here instead of environment variable? For example [Handy](https://github.com/bigbinary/handy) would serve, [Figaro](https://github.com/laserlemon/figaro) or any you think would be a proper tool for this.